examples/acoustic_example.py

```diff
@@ -106,21 +106,26 @@ def damp_boundary(damp, h, nbpml):
             damp[:, :, -(i + 1)] += val
 
 nrec, _ = data.traces.shape
-m = DenseData("m", model.vp.shape, dtype)
+m = DenseData(name="m", shape=model.vp.shape, dtype=dtype)
 m.data[:] = model.vp**(-2)
-u = TimeData("u", m.shape, nt, time_order=t_order, save=False, dtype=m.dtype)
-v = TimeData("v", m.shape, nt, time_order=t_order, save=False, dtype=m.dtype)
+u = TimeData(name="u", shape=m.shape, time_dim=nt, time_order=t_order, save=False, dtype=m.dtype)
```

Comment:
The term `shape` might lead the application developer to think of it as representing the complete shape of the array which includes the time dimension. But I am treating this as the shape only in space, to which a time dimension would be added. Its probably clear to you since you read/wrote the code but do you think the term `shape` makes it clear that its only the space dimensions that contribute to this shape and that the final shape of the array in memory will include another dimension?